img2dataset/downloader.py

```diff
@@ -283,7 +283,7 @@ def download_one_file(url_list):
                 message=f"success={1.0*total_success/total_total:.2f} "
                 message+=f"failed download={1.0*total_failed_to_download/total_total:.2f} "
                 message+=f"failed resize={1.0*total_failed_to_resize/total_total:.2f}"
-                print(message+"\n" , sep=' ', end='', flush=True)
+                tqdm.write(message+"\n" , sep=' ', end='', flush=True)
```

Comment:
Do you really want to flush it constantly? One of the benefits of tqdm is that ensure the progressbar does not slow things down with excessive flushes.

Reply:
Oh,honestly i just swapped print to tqdm.write, I'll change that